axelor-account/src/main/resources/i18n/messages_fr.csv

```diff
@@ -221,6 +221,7 @@
 "Account equivalence","Équivalence de compte",,
 "Account equivalences","Équivalences de compte",,
 "Account filters","Filtres sur le compte",,
+"Account for appropriation to extraordinary depreciations","Compte pour dotation aux amortissements exceptionnels",,

```

Comment:
Why is this line only in this file? It should be also in messages.csv and messages_en.csv